lighthouserc.js

```diff
@@ -1,9 +1,17 @@
 module.exports = {
   ci: {
     collect: {
-      startServerCommand: 'yarn dev',
+      startServerCommand: 'yarn start',
```

Comment:
For anyone wondering about this: @ozer619 says `yarn start` starts the server with settings optimized for production.